config.json

```diff
@@ -1,9 +1,5 @@
 {
   "bootstrap": [
-    "connectora.ringr2.devsca.com",
-    "connectorb.ringr2.devsca.com",
-    "connectorc.ringr2.devsca.com",
-    "connectord.ringr2.devsca.com",
-    "connectore.ringr2.devsca.com"
+      "bucketclient.testing.local:9000"
```

Comment:
I'm not really sure, the tests are started using the `testing.local` stuff. I still wonder why tests broke partly without that change.